src/runtime_src/core/edge/drm/zocl/zocl_bo.c

```diff
@@ -945,3 +945,19 @@ void zocl_clear_mem(struct drm_zocl_dev *zdev)
 
 	mutex_unlock(&zdev->mm_lock);
 }
+
+struct drm_gem_object *zocl_gem_import(struct drm_device *dev,
+				       struct dma_buf *dma_buf)
+{
+	struct drm_gem_object *gem_obj;
+	struct drm_zocl_bo *zocl_bo;
+
+	gem_obj = drm_gem_prime_import_dev(dev, dma_buf, dev->dev);
+	zocl_bo = to_zocl_bo(gem_obj);
+	/* gem_prime_import_sg_table = drm_gem_cma_prime_import_sg_table
```

Comment:
Could you please explain what these comments are about and why they are put here?

Reply:
The comment is trying to hint that drm_gem_cma_prime_import_sg_table() make sure the importing buffer is CMA buffer.
drm_gem_cma_prime_import_sg_table() actually check if buffer is physical contiguous.

Reply:
> The comment is trying to hint that drm_gem_cma_prime_import_sg_table() make sure the importing buffer is CMA buffer.
> drm_gem_cma_prime_import_sg_table() actually check if buffer is physical contiguous.

so maybe just comment explicitly instead of hint? something like DRM will check if buffer is physically contiguous?